runtime/bcutil/attrlookup.h

```diff
@@ -184,9 +186,9 @@ lookupKnownAttribute (register const char *str, register unsigned int len)
   static const signed char lookup[] =
     {
       -1, -1, -1, -1,  0, -1, -1,  1, -1,  2,  3, -1,  4,  5,
-      -1,  6,  7,  8,  9, -1, 10, 11, 12, 13, 14, 15, -1, 16,
-      17, 18, 19, 20, -1, -1, 21, 22, 23, 24, -1, 25, -1, 26,
-      27, -1, -1, -1, -1, 28, -1, -1, 29
+       6,  7,  8,  9, 10, -1, 11, 12, 13, 14, 15, 16, -1, 17,
+      18, 19, 20, 21, -1, -1, 22, 23, 24, 25, -1, 26, -1, 27,
+      28, -1, -1, -1, -1, 29, -1, -1, 30
```

Comment:
Rather than avoiding the use of `size_t`, I suggest `cfreader.c` should be modified to include `stddef.h`.

Reply:
Looking at the build log, it's not `size_t` that's the problem, it's inconsistent use of that type: it should be used as the return type of `attributeHash()` and the local variable `key` which captures that return value.

Reply:
Reverting to (inconsistent) use of `unsigned int` and `int` doesn't seem reasonable to me.